src/style-installer.js

```diff
@@ -304,20 +303,6 @@ function setBodyVisibleStyles(doc) {
   });
 }
 
-/**
- * @param {!Document} doc
- * @noinline
- */
-function renderStarted_(doc) {
-  try {
-    Services.resourcesForDoc(doc.documentElement).renderStarted();
-  } catch (e) {
-    // `makeBodyVisible` is called in the error-processing cycle and thus
-    // could be triggered when runtime's initialization is incomplete which
-    // would cause unrelated errors to be thrown here.
```

Comment:
Is this not true anymore?

Reply:
discussed with @dvoytenko offline, this was because makeBodyVisible was called in `onerror` function, any async exception would cause recursive errors. it's no longer the case, as we now call makeBodyVisibleRecovery in `onerror` now.